tests/Browser/Admin/Bans/BanCreateUpdateTest.php

```diff
@@ -29,7 +29,7 @@ public function test_tooladmin_can_create_and_modify_user_ban()
                 ->visit(route('admin.bans.create'))
                 ->assertDontSee('403')
                 ->type('target', 'UTRS banned user')
-                ->type('reason', 'UTRS public ban reason')
+                ->type('update_reason', 'UTRS public ban reason')
```

Comment:
This is definitely the cause for test errors, creating a ban uses `reason` only. If you want you can make them consistent now (I'd go for reason when updating too instead of using create_reason or something similar when updating) or I can make it consistent when touching ban mode when working on cross-wiki bans.